src/awkward/_connect/numexpr.py

```diff
@@ -124,42 +124,40 @@ def re_evaluate(local_dict=None):
     try:
         compiled_ex = numexpr.necompiler._numexpr_last["ex"]  # noqa: F841
     except KeyError as err:
-        raise ak._v2._util.error(
+        raise ak._util.error(
             RuntimeError(
                 "not a previous evaluate() execution found"
-                + ak._v2._util.exception_suffix(__file__)
+                + ak._util.exception_suffix(__file__)
             )
         ) from err
     names = numexpr.necompiler._numexpr_last["argnames"]
     arguments = getArguments(names, local_dict)
 
     arrays = [
-        ak._v2.operations.to_layout(x, allow_record=True, allow_other=True)
+        ak.operations.to_layout(x, allow_record=True, allow_other=True)
         for x in arguments
     ]
 
     def action(inputs, **ignore):
         if all(
-            isinstance(x, ak._v2.contents.NumpyArray)
-            or not isinstance(x, ak._v2.contents.Content)
+            isinstance(x, ak.contents.NumpyArray)
+            or not isinstance(x, ak.contents.Content)
             for x in inputs
         ):
             return (
-                ak._v2.contents.NumpyArray(
-                    numexpr.re_evaluate(dict(zip(names, inputs)))
-                ),
+                ak.contents.NumpyArray(numexpr.re_evaluate(dict(zip(names, inputs)))),
             )
         else:
             return None
 
-    behavior = ak._v2._util.behavior_of(*arrays)
-    out = ak._v2._broadcasting.broadcast_and_apply(
+    behavior = ak._util.behavior_of(*arrays)
+    out = ak._broadcasting.broadcast_and_apply(
         arrays, action, behavior, allow_records=False
     )
     assert isinstance(out, tuple) and len(out) == 1
-    return ak._v2._util.wrap(out[0], behavior)
+    return ak._util.wrap(out[0], behavior)
 
 
-# ak._v2._connect.numexpr = types.ModuleType("numexpr")
-# ak._v2._connect.numexpr.evaluate = evaluate
-# ak._v2._connect.numexpr.re_evaluate = re_evaluate
+# ak._connect.numexpr = types.ModuleType("numexpr")
+# ak._connect.numexpr.evaluate = evaluate
+# ak._connect.numexpr.re_evaluate = re_evaluate
```

Comment:
It looks like this was a shim to make a subset of the numexpr module public. Now we just have `numexpr.py` instead of `_numexpr.py`, so I *think* I can remove this.